lang/java/compiler/src/main/velocity/org/apache/avro/compiler/specific/templates/java/classic/record.vm

```diff
@@ -163,9 +163,9 @@ public class ${this.mangle($schema.getName())}#if ($schema.isError()) extends or
 #if ($this.hasLogicalTypeField($schema))
   protected static final org.apache.avro.Conversions.DecimalConversion DECIMAL_CONVERSION = new org.apache.avro.Conversions.DecimalConversion();
 #if ($this.useJodaForDateTimeLogicalTypes())
```

Comment:
i think this needs to be refactored to use the new getDateTimeLogicalTypeImplementation() method

Reply:
Good spot!